doc/admin-api.md

```diff
@@ -660,4 +661,36 @@ Config Example:
 }
 ```
 
+## Plugin Metadata
+
+*API*：/apisix/admin/plugin_metadata/{plugin_name}
+
+*Description*: plugin metadata.
+
+> Request Methods:
+
+|Method      |Request URI|Request Body|Description        |
+|---------|-------------------------|--|------|
+|GET      |/apisix/admin/plugin_metadata/{plugin_name}|NULL|Fetch resource|
+|PUT      |/apisix/admin/plugin_metadata/{plugin_name}|{...}|Create resource by plugin name|
+|DELETE   |/apisix/admin/plugin_metadata/{plugin_name}|NULL|Remove resource|
+
+> Request Body Parameters：
+
+An object with at least one property and at most 10 properties.
```

Comment:
wrong comments, please update it

Reply:
And we need to add some doc in `how to develop plugin`, the developer need it:

https://github.com/apache/apisix/blob/4f83f09eebbf10e5b33765c6b331e42fdc3862e3/doc/plugin-develop.md